test/duke/task/TaskListTest.java

```diff
@@ -0,0 +1,62 @@
+package duke.task;
+
+import duke.DukeException;
+import org.junit.jupiter.api.Test;
+
+import java.util.ArrayList;
+
+import static org.junit.jupiter.api.Assertions.*;
+
+class TaskListTest {
+
+    @Test
+    void checkSize_add1Task_success() {
+        TaskList taskList = new TaskList(new ArrayList<Task>());
+        taskList.add( new Todo("todo task description", false) );
```

Comment:
Perhaps you can make these lines of code an attribute of the TaskListTest class?